### ISSUE TITLE
Olympique: Site title appears misaligned

### ISSUE THREAD

**Issue by alaczek** [Automattic/themes#8056]
On the Olympique theme, the site title looks misaligned, if there is no logo image. This is because the group containing the site logo block has a min width set. Ideally, the site title would move to the left and fill the empty space if there is no logo image.

<img width="1357" alt="image" src="https://github.com/user-attachments/assets/eb9bf826-4464-4b34-8af3-cddf329ca3d5">

cc @henriqueiamarino

**Comment by alaczek** [comment]
Ok, I can see now that site title and the little border are aligned with the lines below. This still looks like a visual bug though, since the green posts area is lower - if you have to scroll to see it, the alignment detail is going to be lost.

<img width="1479" alt="image" src="https://github.com/user-attachments/assets/21b817a0-d17e-4875-92ce-78532d74e85d">

**Comment by henriqueiamarino** [comment]
Yes, I get what you mean, @alaczek. The site logo block mirrors the group, with numbers inside the query loops. But once people have no logos, it gets a little off. I may find a solution for it in the future. I'm working on fixing and committing new themes for two weeks, this and the next one.
